ui/narrative/methods/run_maxbin2/display.yaml

```diff
@@ -1,7 +1,7 @@
 #
 # define display information
 #
-name     : MaxBin2 Contig Binning - v2.2.3
+name     : MaxBin2 Contig Binning - v2.2.4
```

Comment:
should be v1.1.0

Reply:
the App name when the wrapped tool itself has a version includes the version of the wrapped tool, not the module version